noweekend-clients/client-mcp/src/main/kotlin/noweekend/client/mcp/McpNotRespondingException.kt

```
@@ -0,0 +1,5 @@
+package noweekend.client.mcp
+
+import java.lang.RuntimeException
+
+class McpNotRespondingException() : RuntimeException()
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)